python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -648,7 +685,7 @@ def _default_run_config_fn(context: ScheduleEvaluationContext) -> CoercibleToRun
                     " to ScheduleDefinition. Must provide only one of the two."
                 )
             elif tags:
-                tags = normalize_tags(tags, allow_reserved_tags=False).tags
+                tags = normalize_tags(tags, allow_reserved_tags=False, warning_stacklevel=5).tags
```

Comment:
unrelated? Probably positive change but generally like isolating stuff

Reply:
Not unrelated-- the use of `@experimental_param` (for `target`) on the constructor adds a level to the stack, so adjusting the `warning_stacklevel` keeps the warning message correct.